### PR TITLE
Added new category for button idea regarding login | signup

### PR DESCRIPTION
**Pull request by Neeraj3508**
Issue:

### CODE REVIEW

**Comment by shahednasser** [comment]
Hi @Neeraj3508 we already have [social login](https://shahednasser.github.io/sbuttons/#social) buttons! If you mean something else, please ellaborate.

**Comment by Neeraj3508** [comment]
we can create rounded versions for that and you only have login button we can also create sign up buttons

**Comment by Neeraj3508** [comment]
i also have designs for some of them

**Comment by shahednasser** [comment]
I don't think the sign up button is necessary, since it would be just changing the text of the buttons. However, I like the rounded buttons idea. Can you change the title (Sign In | Sign Up...) to "Social Buttons, and then the name of the button change it to "rounded buttons" with the description to make them rounded instead? Also, please add "Social Buttons" to the menu list in the beginning of the page.

**Comment by Neeraj3508** [comment]
sure

**Review by Neeraj3508 — COMMENTED** [review]
Changed the things you asked for

File: BUTTON_IDEAS.md [hunk 1]
```diff
@@ -30,6 +31,6 @@ When adding ideas, please add them in the right category (or add a new one if yo
 - [ ] **Retro button**: A button that has a retro style to it.
 - [ ] **Neumorphic button**: A button that has a neumorphic style.
 
- ## Sign In | Sign up  Buttons
+ ## Social Buttons
 
- - [ ] **Google | FaceBook | Github** : A button with login in or sign up to Google or Facebook or Github, etc. 
\ No newline at end of file
+ - [ ] **Rounded  Buttons** : A social button with rounded corners.
```
Comment (shahednasser) [inline BUTTON_IDEAS.md:36]: Change the description to "Add rounded social buttons"

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: BUTTON_IDEAS.md (hunk 1 above)
Comment (Neeraj3508) [reply BUTTON_IDEAS.md:36]: ok

**Comment by shahednasser** [comment]
Great! Thank you.

@all-contributors please add @Neeraj3508 for idea

**Comment by Neeraj3508** [comment]
Thanks @shahednasser for your support throughout my first pull request and looking to contribute more in this repository around hactoberfest.

**Comment by Neeraj3508** [comment]
if you allow i can create these buttons and i have got some more ideas for buttons

**Comment by shahednasser** [comment]
Yes of course @Neeraj3508 ! You can work on anything.

**Pull request merged**